h/models/flag.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import sqlalchemy as sa
+
+from memex.db import types
+
+from h.db import Base
+from h.db.mixins import Timestamps
+
+
+class Flag(Base, Timestamps):
+    """
+    A flag.
```

Comment:
Hahaha :joy:

As much as I like terse documentation, I think it would probably good to assume a little less context here. Something like:

```
"""
A flag representing a user request for moderator attention.

Users can "flag" annotations if they believe that the annotation in question violates the 
content policy of the group or service, or otherwise needs moderator attention.
"""
```